dependency_manager/src/edm_tool/edm.py

```diff
@@ -642,6 +642,20 @@ def parse_workspace_files(cls, workspace_files: list) -> dict:
                         log.error(f"Error parsing yaml of {workspace_file}: {e}")
         return workspace
 
+    @classmethod
+    def parse_workspace_directory(cls, workspace_dir: Path) -> dict:
+        """Parse the given workspace_dir for possible local dependencies"""
+        workspace = {}
+        workspace["local_dependencies"] = {}
+        workspace["workspace"] = workspace_dir.as_posix()
+        for entry in workspace_dir.iterdir():
+            if not entry.is_dir():
+                pass
+            workspace["local_dependencies"][entry.name] = {}
+            workspace["local_dependencies"][entry.name]["git_tag"] = GitInfo.get_branch(entry)
```

Comment:
why not
``workspace["local_dependencies"][entry.name] = {"git_tag": GitInfo.get_branch(entry) }``

Reply:
why not indeed